src/Resources/contao/drivers/DC_Folder.php

```diff
@@ -1126,6 +1133,11 @@ public function move($blnIsAjax=false)
 			// Redirect or reload
 			if (!$objUploader->hasError())
 			{
+				if ($blnIsAjax)
+				{
+					throw new ResponseException(new Response(\Message::generateUnwrapped(), 201));
```

Comment:
Using a `204` status code would only work for single uploads; then you would have to return the URL to the uploaded file in the `Location` header. But `move()` processes multiple uploads at once, so …